### PR TITLE
chore: add redis settings in the event persister service

### PR DESCRIPTION
**Pull request by ono-max**

### CODE REVIEW

File: manifests/bucketeer/charts/event-persister-evaluation-events-kafka/values.yaml [hunk 1]
```diff
@@ -32,6 +32,8 @@ env:
   redis:
     serverName:
     addr:
+    poolMaxIdle: 50
+    poolMaxActive: 200
```
Comment (ono-max) [inline manifests/bucketeer/charts/event-persister-evaluation-events-kafka/values.yaml:36]: Should we change these params?

Comment (masaaania) [reply manifests/bucketeer/charts/event-persister-evaluation-events-kafka/values.yaml:36]: Honestly, I'm not sure how to decide it.
We can experiment with it in the dev, load, or production environment 👍

**Review by masaaania — DISMISSED** [review]
👍 👍

File: manifests/bucketeer/charts/event-persister-evaluation-events-kafka/values.yaml (hunk 1 above)
Comment (cre8ivejp) [reply manifests/bucketeer/charts/event-persister-evaluation-events-kafka/values.yaml:36]: Let's keep both at 50.

**Comment by cre8ivejp** [comment]
Please update the commit message when you merge to: `chore: add redis settings in the event persister service`

**Review by cre8ivejp — APPROVED** [review]
Thank you!

**Pull request merged**